### ISSUE TITLE
configuration for .svelte files

### ISSUE THREAD

**Issue by mvolkmann** [microsoft/vscode-eslint#935]
I haven't been able to get this to flag JavaScript syntax errors in `.svelte` files. However, if I run eslint on the same files from a terminal window it does flag errors. Here is what I tried in my `settings.json` file.

```json
  "eslint.probe": [
    "html",
    "javascript",
    "typescript"
  ],
  "files.associations": {
    "*.svelte": "html"
  },
```

So it should be treating `.svelte` files as the "html" language type and eslint should be running on that language type. Hopefully I am just missing some configuration.

**Comment by mvolkmann** [comment]
I think the issue is that ESLint is not running on the JavaScript inside `.html` files for me. Maybe if I fixed that, it would start working in `.svelte` files too since I'm telling VS Code to treat those as html. What is needed to get the ESLint extension to work on `.html` files?

**Comment by mvolkmann** [comment]
I got this to work! Here is what I did:

`npm install -D eslint eslint-plugin-import eslint-plugin-svelte3`

In `.eslintrc.json`:
```json
{
  "env": {
    "browser": true,
    "es6": true,
    "jest": true,
    "node": true
  },
  "extends": ["eslint:recommended", "plugin:import/recommended"],
  "globals": {
    "cy": "readonly"
  },
  "overrides": [
    {
      "files": ["**/*.svelte"],
      "processor": "svelte3/svelte3"
    }
  ],
  "parserOptions": {
    "ecmaVersion": 2019,
    "sourceType": "module"
  },
  "plugins": ["import", "svelte3/svelte3"],
  "rules": {
    "no-console": "off"
  },
  "settings": {
    "html/html-extensions": [".html", ".svelte"]
  }
}
```

Add the following npm script to `package.json`:

```json
  "lint": "eslint --fix --quiet src --ext .html,.js,.svelte",
```

To run ESLint, enter `npm run lint`.
